ui-tests/src/test/resources/features/customizations/extension-CRUD.feature

```diff
@@ -1,46 +1,29 @@
 @extension-CRUD-test
 Feature: Customization - Extensions CRUD
 
-  @extension-CRUD-clean-application-state
-  Scenario: Clean application state
+  Background: Clean application state
```

Comment:
We probably can't have Background at the start as it's going to break the logic of the folllowing steps.